practice-one/.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Is it possible to create .editorconfig file in root folder instead of putting it in each folders?